### PR TITLE
Documented code

### PR DESCRIPTION
**Pull request by ReflxctionDev**
* Documented a couple of addon classes. Documented all/most command classes. Documented event classes. Documented most config classes (except those which I didn't know what they do or what are they for), and on more classes

* Added **sources** and **javadoc** building methods for Gradle. Those will allow any developer who has Hyperium as a dependency (e.g addon environment) to browse javadocs from their IDE rather than showing it as decompiled .class files. Those can be removed from **build.gradle** if this is not desired.

I tend to document more classes later, on more runs.

### CODE REVIEW

**Comment by RDIL** [comment]
agree

File: build.gradle [hunk 1]
```diff
@@ -156,3 +156,19 @@ jar {
     }
 }
 
+// So the user may be able to browse Javadocs from an addon environment
+
+//noinspection GroovyAssignabilityCheck,GrUnresolvedAccess
+task sourcesJar(type: Jar, dependsOn: classes) {
```
[the remark this answers is not in the document]
Comment (ReflxctionDev) [reply build.gradle]: So is there anything I should remove other than the sourcesJar task? Thanks for your reminders, I'll note them next time.

File: src/main/java/cc/hyperium/commands/defaults/CommandDebug.java [hunk 2]
```diff
@@ -32,7 +33,7 @@
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
 
-import java.awt.Toolkit;
+import java.awt.*;
```
Comment (RDIL) [inline src/main/java/cc/hyperium/commands/defaults/CommandDebug.java]: This would make @asbyth unhappy

**Comment by RDIL** [comment]
This PR is now stale.

**Pull request closed without merging**